src/System.Net.Http/tests/FunctionalTests/HttpProtocolTests.cs

```diff
@@ -27,17 +29,21 @@ public class HttpProtocolTests : HttpClientTest
         [InlineData("HTTP/1.1 500 Internal Server Error", 500, "Internal Server Error")]
         [InlineData("HTTP/1.1 555 we just don't like you", 555, "we just don't like you")]
         [InlineData("HTTP/1.1 600 still valid", 600, "still valid")]
-        // TODO #24713: The following pass on Windows on .NET Core but fail on .NET Framework.
-        //[InlineData("HTTP/1.1 200      ", 200, "")]
-        //[InlineData("HTTP/1.1 200      Something", 200, "Something")]
-        //[InlineData("HTTP/1.1\t200 OK", 200, "OK")]
-        //[InlineData("HTTP/1.1 200\tOK", 200, "OK")]
-        //[InlineData("HTTP/1.1 200", 200, "")]
-        //[InlineData("HTTP/1.1 200\t", 200, "")]
-        //[InlineData("HTTP/1.1 200 O\tK", 200, "O\tK")]
-        //[InlineData("HTTP/1.1 200 O    \t\t  \t\t\t\t  \t K", 200, "O    \t\t  \t\t\t\t  \t K")]
-        //[InlineData("HTTP/1.1 999 this\ttoo\t", 999, "this\ttoo\t")]
         public async Task GetAsync_ExpectedStatusCodeAndReason_Success(string statusLine, int expectedStatusCode, string expectedReason)
+        {
+            await GetAsyncSuccessHelper(statusLine, expectedStatusCode, expectedReason);
+        }
+
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "The following pass on Windows on .NET Core but fail on .NET Framework.")]
```

Comment:
I assume they pass also on Linux - can you remove the 'Windows' note?

Reply:
I will need to see the CI results. (I didn't verify on Linux)